packages/components/src/mobile/focal-point-settings-panel/index.native.js

```diff
@@ -56,7 +56,7 @@ const FocalPointSettingsPanelMemo = memo(
 				</NavBar>
 				<FocalPointPicker
 					focalPoint={ draftFocalPoint }
-					onChange={ setPosition }
+					onChange={ useCallback( () => setPosition(), [] ) }
```

Comment:
I tested this PR on a iPhone SE and Samsung Galaxy S20. It seems the coordinates are no longer persisted after dragging the focal point handle. After releasing the handle, it "snaps" back to the initial position. 

I believe the issue is that the `onChange` arguments are not passed through to the new memoized `setPosition`. The easiest solution might be the following, which does fix the issue from my testing. I do not believe the wrapping anonymous function is not all that necessary. 

```suggestion
					onChange={ useCallback( setPosition, [] ) }
```